src/Congo.WebApi/Data/Models/Product.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Congo.WebApi.Data.Models
+{
+    public class Product
+    {
+        [Key]
+        public string Id { get; init; } = Guid.NewGuid().ToString();
```

Comment:
Agree with Thorsten, if we want Guid style IDs, this should be `Guid` prop, not a string. My only gripe with them is that they are very long and take up a lot of URL space, so an alternative worth considering is the "YouTube" style URL-safe base64 IDs. Doable, but perhaps overkill.

I will not die on this hill, so unless you feel very strongly about making a YouTube style identifier, Guid is fine.